roles/provision_vcenter/templates/config_parts/ceip.j2

```diff
@@ -0,0 +1,31 @@
+"ceip": {
+    "description": {
+        "__comments": [
+            "++++VMware Customer Experience Improvement Program (CEIP)++++",
```

Comment:
Can you add a link to the original description so if and when it will be updated we will update it here as well?

Reply:
I updated this section (and the others you mentioned) to refer to a README.md in the templates directory. I listed possible help commands and linked to version of vmware documentation that should be helpful when working with these config files.
Let me know if that helps clear things up, or if I should keep all the documentation in the templates. The comments in the templates currently came from VMWare, so I thought I should leave them there.